unicodetools/org/unicode/propstest/UnicodePropertyPartition.java

```diff
@@ -0,0 +1,90 @@
+package org.unicode.propstest;
+
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.Map.Entry;
+
+import org.unicode.cldr.util.Counter;
+import org.unicode.props.IndexUnicodeProperties;
+import org.unicode.props.PropertyStatus;
+import org.unicode.props.PropertyStatus.PropertyOrigin;
+import org.unicode.props.PropertyType;
+import org.unicode.props.UcdProperty;
+
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Multimap;
+import com.google.common.collect.Ordering;
+import com.google.common.collect.TreeMultimap;
+import com.ibm.icu.dev.util.UnicodeMap;
+import com.ibm.icu.text.UnicodeSet;
+
+public class UnicodePropertyPartition {
+    static IndexUnicodeProperties UnicodeProperties = IndexUnicodeProperties.make();
+    static Comparator<UnicodeSet> SIZE_FIRST = new Comparator<UnicodeSet>() {
+	@Override
```

Comment:
Would you mind fixing funky indentation in this file?

Reply:
I will redo the formatting. I held off so that it wouldn't introduce spurious differences.